server/system/schedulers/tempFilesCleanup.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
+const path = require('path')
+const schedule = require('node-schedule')
+
+const Logger = require('../../log/log').getLogger('TempFilesCleanup')
+
+const DateUtils = require('../../../common/dateUtils')
+
+const tempFolder = '/tmp/arena-upload'
+
+const initSchedule = () =>
+  // execute the cron job every day at 2AM
+  schedule.scheduleJob('0 2 * * *', async () => {
+    // cleanup temp files older than 6 hours
+    await cleanupTempFiles(6)
+  })
+
+const cleanupTempFiles = async (olderThanHours = null) => {
+  Logger.debug('Cleaning up temp files')
+
+  let count = 0
+  const now = new Date()
+
```

Comment:
I don't really see the need to call the function visit temp files with the visitor to that just delete the file. Why not to do everything in this method?

Reply:
ok